Numerical Descriptives in R.Rmd

```diff
@@ -28,7 +28,7 @@ One of the first steps that you might take once you have some data in R is to lo
 
 ### Example Dataset: Iris
 
-In the examples for this tutorial, we will use the \texttt{iris} dataset provided in \texttt{MASS} package in R. 
+In the examples for this tutorial, we will use the `iris` dataset provided in `datasets` package in R.
```

Comment:
When I viewed this locally, the word `iris` wasn't showing up so I removed the `\texttt`

Also, I *think* iris and mtcars are in a package called *datasets*. But maybe *MASS* is related?

Reply:
Thanks for pointing out! Resolved.